src/TensorFlowNET.Core/Variables/variables.py.cs

```diff
@@ -155,7 +157,10 @@ public static Operation _safe_initial_value_from_op(string name, Operation op, D
 
         public static Tensor global_variables_initializer()
         {
-            throw new NotImplementedException();
```

Comment:
Should we use `tf.compat.v1.global_variables_initializer()`?

Reply:
I think so. Thanks!

Reply:
Followed the advice